### DISCUSSION TITLE
Checkbox code snippet installation error

### DISCUSSION THREAD

**Discussion by diep89** [chakra-ui/chakra-ui#9404] [General]
### Description

When trying to generate a local checkbox component through the provided snippet, an ark module resolution error pops up.

![image](https://github.com/user-attachments/assets/0f0a8bee-81fe-4cbe-acdd-2f1de26eae99)

Tried Node versions `20.11.1` and current latest `22.12.0` with no luck.

### Link to Reproduction

https://link-not-necessary

### Steps to reproduce

1. Setup Chakra v3 project
2. On terminal, run `npx @chakra-ui/cli snippet add checkbox`
3. See error

### Chakra UI Version

3.2.3

### Operating System

- [ ] macOS
- [ ] Windows
- [X] Linux

**Comment by alanbosco003** [comment]
I'm also gettig something similer. when installing snippet.

**Comment by segunadebayo** [comment]
Do you have `@chakra-ui/react` installed?

If so, what package manager do you use?

**Comment by segunadebayo** [comment]
Make sure `@chakra-ui/react` is installed. The snippet no longer imports `@ark-ui`.
